style.css

```diff
@@ -0,0 +1,75 @@
+/*https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Backgrounds_and_Borders/Using_multiple_backgrounds*/
+
+body {
+    background-image: url(https://mdn.mozillademos.org/files/11307/bubbles.png), linear-gradient(to right, rgba(30, 75, 115, 1), rgba(255, 255, 255, 0));
+    background-repeat: no-repeat, no-repeat, no-repeat;
+    background-position: bottom right, left, right;
+}
+
+ul {
+    list-style-type: none;
+}
+
+.month {
+    margin-top: -16px;
+    padding: 25px;
+    width: 100%;
+    text-align: center;
+}
+
+.month ul {
+    margin: 0;
+    padding: 0;
+}
+
+.month ul li {
+    font-size: 20px;
+    text-transform: uppercase;
+}
+
+.month .prev {
+    float: left;
+    padding-top: 10px;
+    font-size: 75px;
+}
+
+.month .next {
+    float: right;
+    padding-top: 10px;
+    font-size: 85px;
+}
+
+.wWeekdays {
+    margin: 0;
+    padding: 10px 0;
+}
+
+.wWeekdays li {
+    display: inline-block;
+    width: 13.2%;
+    text-align: center;
+    margin-right: 15px;
+    font-size: 25px;
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif
+}
+
+.days {
+    margin: 0;
+    padding: 20px 0;
+}
+
+.days li {
+    list-style-type: none;
+    display: inline-block;
+    width: 13%;
+    text-align: center;
+    margin-bottom: 10px;
+    margin-left: 15px;
+    font-size: 12px;
+    padding-top: 37px;
+    padding-bottom: 30px;
+    background: rgb(5, 0, 36);
+    background: linear-gradient(324deg, rgba(5, 0, 36, 0) 23%, rgba(96, 149, 237, 1) 94%);
```

Comment:
https://cssgradient.io/ i forgot to add the link to the style sheet in a comment